src/scalar.jl

```diff
@@ -35,24 +35,24 @@ has_mul!(::AbstractSciMLScalarOperator) = true
 islinear(::AbstractSciMLScalarOperator) = true
 has_adjoint(::AbstractSciMLScalarOperator) = true
 
-Base.:*(α::AbstractSciMLScalarOperator, u::AbstractVecOrMat) = convert(Number, α) * u
```

Comment:
since `ScalarOperator` doesn't depend on size, it can be applied to all `AbstractArray`s